StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -54,10 +55,12 @@ class EmbeddedPaymentMethodsView: UIView {
          shouldShowApplePay: Bool,
          shouldShowLink: Bool,
          savedPaymentMethodAccessoryType: RowButton.RightAccessoryButton.AccessoryType?,
-         mandateProvider: MandateTextProvider) {
+         mandateProvider: MandateTextProvider,
+         shouldShowMandate: Bool = true) {
         self.appearance = appearance
         self.selection = initialSelection
         self.mandateProvider = mandateProvider
+        self.shouldShowMandate = shouldShowMandate
```

Comment:
I'd think we're going to pass this thing a `Configuration` eventually, so we'll probably get rid of this and just reach directly for `configuration.embeddedViewDisplaysMandateText` but we can change it when we get there.